library/pom.xml

```diff
@@ -372,9 +377,39 @@
 							</execution>
 						</executions>
 					</plugin>
+
 				</plugins>
 			</build>
 		</profile>
+		<profile>
```

Comment:
Thanks @jonespm, yeah I'll have a look at doing as you suggest, I'm not sure appending of properties works.